digitalpublicgoods/vrapeutic.json

```diff
@@ -0,0 +1,98 @@
+    "bestPracticesList": [
+      "Test of Variables of Attention (TOVA)",
+      "Health Insurance Portability and Accountability Act (HIPPA)",
+      "Principles for Digital Development"
+    ]
+  },
+  "doNoHarm": {
+    "preventHarm": {
+      "stepsToPreventHarm": "Yes",
+      "additionalInfoMechanismProcessesPolicies": "Equipment-Related Risks \n - Possible overheating of VR headsets: We strictly adopt headsets that have an accessible document for Declaration of Conformity, e.g., CE certification. Our Terms of Use also indicate clearly that we don’t permit our clients to use non-CE-certified headsets. Also, our user instructions explicitly mention the maximum duration of allowable continuous operation on a VR headset with a single user/child which specifically addresses the overheating concern. Finally, we recommend to our clients a wide spectrum of alternatives, with regards to VR headsets, on which we test all our VR modules prior to product delivery. \n- Hardware-Software compatibility: We should always make sure that the developed software is compatible with the processor of the headset in terms of computational efficiency. Otherwise, the content would be poorly rendered with noticeable flickering, which impacts the user experience remarkably and biases the results. Since we develop the curriculum in house, we recommend to the clients the headsets on which the curriculum is tested by our team. Moreover, to mitigate this risk, we provide our product on a wide variety of headsets, with a wide range of costs, mainly to give the client the flexibility of choice according to their budget and accepted experience quality. \n\nOperational Risks \n- Potential dizziness effects on users: Our safety policy and user instructions strictly mentions that our system is not allowed to be used without a physically-existing supervisor. If the sessions are taking place on-premises so the specialist is the supervisor, and if the session is taking place at home, so the specialist is supervising remotely but a guardian is accompanying the child physically. This is mainly to remove the headset in case dizziness has been observed. Also, our user instructions explicitly mentions the maximum duration of allowable usage on a VR headset with a single user/child which addresses the dizziness concern. Finally, we only adopt VR headsets with explicit dizziness instructions for the users including Oculus, Pico, and Gear VR. \n- Potential habituation if not enough scenarios are developed and used: Our product plan takes into consideration the development of an ever-growing library of modules that takes into account the diversity of interests, social scenarios, acceptable difficulty levels, and the age range. This minimizes the possibility of habituation."
+    },
+    "dataPrivacySecurity": {
+      "collectsPII": "Yes",
+      "typesOfDataCollected": [
+        "Neuro-markers such as impulsivity scores, attention performance indices, in addition to others."
```

Comment:
"in addition to others" is not a valid answer here, please specify. Just by reading at their privacy policy, one learns that they collect usernames, emails and passwords, location, phone number (optional), payment information, etc...

Reply:
Resolved in commit bba5864